oresat_configs/oresat0_5/beacon.yaml

```diff
@@ -1,4 +1,12 @@
 revision: 0
+ax25:
+  dest_callsign: SPACE
+  dest_ssid: 0 # primary station
+  src_callsign: KJ7SAT
+  src_ssid: 11 # balloons, aircraft, spacecraft, etc
+  control: 0x3 # ui-frame
+  pid: 0xf0 # no L3 protocol
+  command: true
```

Comment:
I suggest having this be a response packet instead of a command packet. Or `command: false`